product/wordpress/wordpress.go

```diff
@@ -0,0 +1,74 @@
+package wordpress
+
+import (
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/vulncheck-oss/go-exploit/config"
+	"github.com/vulncheck-oss/go-exploit/output"
+	"github.com/vulncheck-oss/go-exploit/protocol"
+	"github.com/vulncheck-oss/go-exploit/random"
+)
+
+var LoginPath = `wp-login.php`
+
+// Turns `net/http` []*Cookie into a string for adding to the Cookie header.
+func createCookieHeader(cookies []*http.Cookie) string {
```

Comment:
This seems fairly generic. Care to move it to protocol? Also could probably get a very easy unit test.

Reply:
I'm not entirely pleased with how this parses and I think could use some polishing to be closer to real browsers, but it should work for now and I added some tests.